linux_pipeline/Jenkinsfile_azure_lis_hotfix_pipeline

```diff
@@ -83,13 +83,35 @@ def GetErrataKernelLink(kernelVersion) {
 
 def CheckerrataLink(errataLink) {
     def status = sh (returnStatus: true, script: """
-        wget ${errataLink}
+        wget -q ${errataLink}
         """ )
     if ( status != 0) {
         throw new Exception("${errataLink} is unreachable.")
     }
 }
 
+def DownloadStashKernelPackages(storage, sasKey, azcopyLink, kernelVersion)
+{
+    node('jenkins-meta-slave') {
+        sourceDir = "https://${storage}.blob.core.windows.net/kernel/${kernelVersion}/"
+        def status = sh (returnStatus: true, script: """#!/bin/bash
+        wget -q -O azcopy.tar.gz ${azcopyLink}
+        tar -xf azcopy.tar.gz
+        azcopy_linux_amd64*/azcopy copy '${sourceDir}${sasKey}' . --recursive=true
+        tar -C ./${kernelVersion}/ -cvf ${kernelVersion}.tar.gz \$(ls ./${kernelVersion} | xargs)
+        """)
+        if ( status != 0) {
+            throw new Exception("Failed to download ${kernelVersion}" )
+        }
+        //stash the kernel packages
+        stash includes: "${kernelVersion}.tar.gz", name: "Package-${kernelVersion}-stash"
+        sh (returnStatus: true, script: """#!/bin/bash
```

Comment:
instead of sh command here, just use cleanWs()

Reply:
done